source/vir/src/def.rs

```diff
@@ -66,6 +66,7 @@ const SUBST_RENAME_SEPARATOR: &str = "$$";
 const KRATE_SEPARATOR: &str = "!";
 const PATH_SEPARATOR: &str = ".";
 const PATHS_SEPARATOR: &str = "/";
+const IMPL_SEPARATOR: &str = "/";
```

Comment:
Is it OK for the IMPL_SEPARATOR to be the same as the PATHS_SEPARATOR? the impl name can be part of a path, right?

Reply:
Good point.  I changed IMPL_SEPARATOR to "&/".